src/utils/index.js

```diff
@@ -18,28 +18,39 @@ const CLIENT_URL = process.env.REACT_APP_STAGE === 'production'
   : 'http://localhost:3000/'
 
 export const AWS_CONFIG = {
-  aws_project_region: 'us-west-2',
-  aws_cognito_identity_pool_id: 'us-west-2:0bfef155-88d4-40cb-9805-de9d366d6650',
-  aws_cognito_region: 'us-west-2',
-  aws_user_pools_id: 'us-west-2_w0R176hhp',
-  aws_user_pools_web_client_id: '5tc2jshu03i3bmtl1clsov96dt',
-  oauth: {
-    domain: 'auth.ubcbiztech.com',
-    scope: [
-      'phone',
-      'email',
-      'openid',
-      'profile',
-      'aws.cognito.signin.user.admin'
-    ],
-    redirectSignIn: CLIENT_URL + 'login-redirect/',
-    redirectSignOut: CLIENT_URL,
-    responseType: 'code'
+  Auth: {
+    aws_project_region: 'us-west-2',
+    aws_cognito_identity_pool_id: 'us-west-2:0bfef155-88d4-40cb-9805-de9d366d6650',
+    aws_cognito_region: 'us-west-2',
+    aws_user_pools_id: 'us-west-2_w0R176hhp',
+    aws_user_pools_web_client_id: '5tc2jshu03i3bmtl1clsov96dt',
+    oauth: {
+      domain: 'auth.ubcbiztech.com',
+      scope: [
+        'phone',
+        'email',
+        'openid',
+        'profile',
+        'aws.cognito.signin.user.admin'
+      ],
+      redirectSignIn: CLIENT_URL + 'login-redirect/',
+      redirectSignOut: CLIENT_URL,
+      responseType: 'code'
+    },
+    federationTarget: 'COGNITO_USER_POOLS',
+    identityPoolId: 'us-west-2:0bfef155-88d4-40cb-9805-de9d366d6650',
```

Comment:
These had to be added again for some reason.
On another note, shouldn't these cognito IDs be in the env file?